### PR TITLE
Update 0080-SPOT-product_builtin_spot.md

### PR DESCRIPTION
**Pull request by VanithaVega**
Changed the Network parameter to SpotMarketTradingEnabled

### CODE REVIEW

**Review by cdummett — DISMISSED** [review]

File: protocol/0080-SPOT-product_builtin_spot.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ When trading Spot products, parties can only use assets they own - there is no l
 
 ## 2. Network Parameter
 
-1. `spot_trading_enabled`: parameter defines whether markets using Spot products are enabled on the network.
+1. `SpotMarketTradingEnabled`: parameter defines whether markets using Spot products are enabled on the network.
```
Comment (gordsport) [inline protocol/0080-SPOT-product_builtin_spot.md]:
```suggestion
1. `limits.markets.proposeSpotEnabled`: parameter defines whether markets using Spot products are enabled on the network.
```

@ze97286 - we have reviewed this in the protocol design meeting and suggest the net param should be named as suggested here

Comment (ze97286) [reply protocol/0080-SPOT-product_builtin_spot.md]: proposeSpotEnabledFrom, is this a date?

Comment (gordsport) [reply protocol/0080-SPOT-product_builtin_spot.md]: @davidsiska-vega - I think this was a boolean, but the suggested name implies a date. Please can you confirm if this should have been `limits.markets.proposeSpotEnabled` instead? 

Current markets (futures?) has: `limits.markets.proposeEnabledFrom`

Comment (davidsiska-vega) [reply protocol/0080-SPOT-product_builtin_spot.md]: Good point! I think it should be `limits.markets.proposeSpotEnabled` and it should just be one boolean. 

I don't think we want to bother with the "enabled from" date.

Comment (gordsport) [reply protocol/0080-SPOT-product_builtin_spot.md]: suggestion updated - can be reviewed and committed in todays meeting

**Review by ze97286 — APPROVED** [review]

**Review by Jiajia-Cui — APPROVED** [review]
LGTM

**Pull request merged**